.eslintrc.json

```diff
@@ -10,6 +10,8 @@
     // "@fullstacksjs/eslint-config/graphql"
   ],
   "rules": {
-    "jest/no-deprecated-functions": "off"
+    "jest/no-deprecated-functions": "off",
+    "no-warning-comments": "off",
```

Comment:
Why?

Reply:
because I get warnings for writing FIXMEs and that's something that I don't want to ignore everytime I write it